assets/scss/main.scss

```diff
@@ -1,6 +1,6 @@
 // /////////////////////////////////////////////////////////////////////// Fonts
 // -----------------------------------------------------------------------------
-$fontAssetPath: '~assets/fonts';
+$fontAssetPath: '~static/fonts';
```

Comment:
Nice, this works great. Now the font files will always live in one place, unaffected by the static site compilation.